src/test/unit/osd_unittest.cc

```diff
@@ -1015,6 +1015,8 @@ extern "C" {
 
     uint16_t getRssi(void) { return rssi; }
 
+    uint8_t getRssiPercent(void) { return rssi * 100 / 1024; }
```

Comment:
Originally it was `(getRssi() * 100) / 1023` and you are using `(getRssi() * 100) / 1024`

Why is that?

Reply:
There was a mix of 1023 and 1024 as the divisor used in scaling down the 'raw' RSSI value. I have standardised this to 1024, which is the correct divisor for a 10 bit integer.